frontend/src/Components/MainContainer/MainContainer.tsx

```diff
@@ -9,6 +9,12 @@ import { ICategory, IFullCategory } from "../../Types/ICategory";
 import { IAnswer } from "../../Types/IAnswer";
 import { generateFullCategories } from "../../Helpers/helper";
 import QuestionModal from "../QuestionModal/QuestionModal";
+import { IPlayerBoxProps } from "../PlayerBox/IPlayerBoxProps";
+
+type Player = {
+  name: string;
+  points: number;
+};
 
 const MainContainer = () => {
```

Comment:
The main container should definetly be refactored. We should synchronise this with Vesela, but if it is possible start separating the main container into logical portions. For example check if we can separate the entirety of the player logic into a PlayerContainer component (,which should be stateful). Then through props the PlayerContainer can communicate with the MainContainer (for example the main container could pass the information about the questions). That's not a guide, just an idea, which you can use to conceptualize different subparts of the main container. There is no need to go ballistic and encapsulate everything into separte containers, but the player logic is quite large and could be a good starting point